lib/src/webcrypto/webcrypto.ecdh.dart

```diff
@@ -14,6 +14,42 @@
 
 part of 'webcrypto.dart';
 
+/// Elliptic Curve Diffie-Hellman (ECDH) is a key agreement protocol that allows
+/// two parties to establish a shared secret over an insecure channel.
+/// 
+/// An [EcdhPrivateKey] instance represents a private key that can be used to
+/// derive a shared secret with a public key.
```

Comment:
Let's always start with a short summary that just says with something is and what it's for.

```suggestion
/// ECDH private key for deriving a shared secret.
///
/// Elliptic Curve Diffie-Hellman (ECDH) is a key agreement protocol that allows
/// two parties to establish a shared secret over an insecure channel.
/// An [EcdhPrivateKey] holds a private key that can be used to derive a
/// shared secret giving the public key from a different key pair.
```

Reply:
Noted